### DISCUSSION TITLE
Can't place widgets on screen

### DISCUSSION THREAD

**Discussion by orion14794** [Archfx/duo-de#108] [Q&A]
Well the title says it all. Widgets can't be placed on home screen.

**Comment by sir86** [comment]
its well know bug #84 

if you install 3rd party widgets from store some of them work, like some clocks for example but mostly not.

**Comment by Archfx** [comment]
This issue is still in A15-Alpha releases. Instead of relying on the system clock widget, I recommend this [clock](https://github.com/BlackyHawky/Clock) app.
